### ISSUE TITLE
Cannot build on Mac with Apple Silicon

### ISSUE THREAD

**Issue by evs38** [huggle/huggle3-qt-lx#353]
Tried to build Huggle on Mac with M1 and get this error:
```
-- Performing Test FLAG_WEXTRA
-- Performing Test FLAG_WEXTRA - Success
CMake Error at extensions/extension-mass-delete/CMakeLists.txt:26 (find_package):
  By not providing "FindQt5WebEngine.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "Qt5WebEngine", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5WebEngine" with
  any of the following names:

    Qt5WebEngineConfig.cmake
    qt5webengine-config.cmake

  Add the installation prefix of "Qt5WebEngine" to CMAKE_PREFIX_PATH or set
  "Qt5WebEngine_DIR" to a directory containing one of the above files.  If
  "Qt5WebEngine" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
See also "/Users/evs/src/huggle3-qt-lx/release/CMakeFiles/CMakeOutput.log".
See also "/Users/evs/src/huggle3-qt-lx/release/CMakeFiles/CMakeError.log".
```

**Comment by benapetr** [comment]
Hello, which version of Qt is this and how was it installed? If you used Qt installer, make sure to use 5.x Qt (not 6.x) and to select Web Engine (chromium) support.

**Comment by benapetr** [comment]
btw I am wondering if this isn't only related to that extension mass-delete, maybe try building huggle without extensions first (wipe build folder and run configure without --extension)

**Comment by evs38** [comment]
Looks like it was fixed on Qt 5.15.6
